website/lib/types.ts

```
@@ -354,6 +354,153 @@ export interface HealthResponse {
+  from: string;
+  to: string;
+  type: string;
+  properties?: Record<string, unknown>;
+}
+
+export interface GraphPath {
+  path: string[];
+  length: number;
+}
+
+export interface GetGraphStatsRequest {}
```

Biome (2.3.13)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)